Android/app/src/main/java/fashiome/android/models/Product.java

```diff
@@ -259,11 +259,6 @@ public void setProductSKU(String productSKU) {
      */
     public Product() {
         super();
-        this.setProductRating(new Random().nextInt(4) + 1);
-        this.setNumberOfFavorites(new Random().nextInt(500) + 100);
-        this.setNumberOfReviews(new Random().nextInt(20) + 1);
-        this.setNumberOfViews(new Random().nextInt(1000) + 100);
-        this.setNumberOfRentals(new Random().nextInt(500) + 100);
     }
 
```

Comment:
Setting the values in the constructor of parse object caused below exception and so the none of the PRODUCTs were received in the parse query. Below are the details

java.lang.IllegalStateException: A ParseObject subclass default constructor must not make changes to the object that cause it to be dirty.

http://stackoverflow.com/questions/25896462/parseobject-subclass-cannot-make-changes-in-constructor